install_resources_no_source_files/after/Pods/Pods.xcodeproj

```diff
@@ -73,6 +73,17 @@ Targets:
           SDKROOT: macosx
           SKIP_INSTALL: 'YES'
         Base Configuration: Pods-SampleApp.release.xcconfig
+- res:
```

Comment:
this is correct now. `res` is a vendored library target so it gets an aggregate target.